build.gradle

```diff
@@ -154,10 +149,10 @@ bintray {
 }
 
 def versions = [
-        commonsLang3         : '3.9',
+        commonsLang3         : '3.11',
         junit                : '4.13',
-        lombokVersion        : '1.16.18',
-        mockito              : '3.3.3',
+        lombokVersion        : '1.18.12',
```

Comment:
ooooo, be careful with that.

I remember I left it 1.1.6.18 for a reason.

I don't remember the reason, but I guess it was something related to payment and issue between jackson <-> lombok. 

Please make sure this will be tested very well by Vivek before you merge it. Especially create a case journey + payment.

Reply:
Valid point 👍  Reverted version increase